src/internal/kopia/upload.go

```diff
@@ -148,6 +148,10 @@ func (cp *corsoProgress) FinishedFile(relativePath string, err error) {
 			Label(fault.LabelForceNoBackupCreation))
 
 		return
+	} else if info.Modified().IsZero() {
+		cp.errs.AddRecoverable(ctx, clues.New("mod time of zero").
```

Comment:
Nit: "of zero" is kinda confusing.
```suggestion
		cp.errs.AddRecoverable(ctx, clues.New("zero-valued mod time").
```